src/SweetAlert.js

```diff
@@ -73,191 +73,195 @@ function warningRemoved(props) {
   })
 }
 
-export default class SweetAlert extends Component {
-  /* eslint-disable react/no-unused-prop-types */
-  static propTypes = {
-    // sweetalert option
-    title: PropTypes.string.isRequired,
-    text: PropTypes.string,
-    swal: PropTypes.func,
-    type: PropTypes.oneOf(['warning', 'error', 'success', 'info', 'input']),
-    customClass: PropTypes.string,
-    showCancelButton: PropTypes.bool,
-    showConfirmButton: PropTypes.bool,
-    confirmButtonText: PropTypes.string,
-    confirmButtonColor: PropTypes.string,
-    confirmButtonClass: PropTypes.string,
-    cancelButtonText: PropTypes.string,
-    cancelButtonClass: PropTypes.string,
-    reverseButtons: PropTypes.bool,
-    buttonsStyling: PropTypes.bool,
-    imageUrl: PropTypes.string,
-    html: PropTypes.bool,
-    animation: PropTypes.oneOfType([
-      PropTypes.bool,
-      PropTypes.oneOf(['pop', 'slide-from-top', 'slide-from-bottom']),
-    ]),
-    // inputType: PropTypes.oneOf(ALLOWS_INPUT_TYPES),
-    inputPlaceholder: PropTypes.string,
-    inputValue: PropTypes.string,
-    showLoaderOnConfirm: PropTypes.bool,
+export const asReactComponent = swalInstance => (
```

Comment:
@alex-shamshurin the class is now wrapped in an anonymous function that passes in a swal instance. If I don't indent these lines then the code now longer follow the formatting rules. I am implementing the change discussed above.

Reply:
Hmm.. I thought diffs on GitHub ignored white-space changes, but I guess not, or at least not any more.

@alex-shamshurin If you use a local Git GUI like SourceTree you can ignore white-space changes while viewing the diff